verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -24,7 +24,30 @@
 namespace verilog {
 namespace kythe {
 
-void KytheFactsExtractor::Visit(const IndexingFactNode& node) {
+void KytheFactsExtractor::ExtractKytheFacts(const IndexingFactNode& root) {
+  CreatePackageScopes(root);
```

Comment:
This won't handle package imported from other files. Only those defined in the same file, right?

This is intentional, IIUC. (for now, until we have support for processing multiple files)

Or am I missing something?

Reply:
Yes this is intended for single files but in case the same KyhteFactsExtractor object is used for processing other files the packages would still be there.
But for now this works fine for single files.